io/net/virt-net/network_virtualization.py

```diff
@@ -482,6 +521,17 @@ def test_remove(self):
                 self.fail("lshwres still lists the Network virtualized device \
                            after remove operation")
 
+    def run_command_vios(self, param):
```

Comment:
@smruti77 For run_command, get the object as an input, and use only one run_command() function.